topostats/entry_point.py

```diff
@@ -52,6 +52,12 @@ def create_parser() -> arg.ArgumentParser:
         required=False,
         help="Path to a YAML configuration file for summary plots and statistics.",
     )
+    process_parser.add_argument(
```

Comment:
Just commenting here to make sure you're aware that you haven't added this argument for the legacy `run_topostats` entry point. I agree with this, as I think we should discourage use of the old way, but wanted to make sure this was intentional 👍

Reply:
Yep, trying to steer people to the newer approach. I'd almost be inclined to remove the older `run_topostats` entry point to do so. Captured in #712